src/translation-assistant.js

```diff
@@ -589,6 +594,16 @@ class TranslationAssistant {
                 }
             }
 
+            // Translate items that are only math even if there's no template.
+            // TODO(kevinb) handle multiple non-nl_text items
```

Comment:
I've separated and moved this code below the code that translates strings according to a template so that `maybeTranslateMath` can be applied to math-only strings. If the template is not available, we'll apply only `translateMath()` here.